### DISCUSSION TITLE
Is there a way to start notification server?

### DISCUSSION THREAD

**Discussion by tukusejssirs** [ErikReider/SwayNotificationCenter#182] [General]
I am using Arch Linux, Sway and SwayNC.

I am experiencing [this issue](https://bbs.archlinux.org/viewtopic.php?id=236020), where the [solution](https://bbs.archlinux.org/viewtopic.php?pid=1778642#p1778642) is to run [standalone notification server](https://wiki.archlinux.org/title/Desktop_notifications#Notification_servers).

Now, I have no `/usr/lib/notification-daemon-1.0/notification-daemon` file (not even `/usr/lib/notification-daemon-1.0/` folder), which is provided by [`notification-daemon`](https://archlinux.org/packages/community/x86_64/notification-daemon/) package.

Now, I could simply install `notification-daemon` and configure the standalone notification server, however, I have no idea if that is the right way to do it when using Sway + SwayNC.

Could you shed some light into this please? :pray:

Anyway, I suggest to enable [GitHub Discussions](https://github.com/features/discussions), as sometimes users have some questions (like this) and other stuff that are not code/bug related.

**Comment by ErikReider** [comment]
```
exec swaync
```

In your sway config. Check the README for more information :)

**Comment by tukusejssirs** [comment]
I have `exec swaync` in my Sway `config` file since I started using SwayNC, therefore this does not fix my issue.

Note that SwayNC works as expected, however, some apps (like Slack installed from AUR using `slack-desktop`) fails with that error. After restarting Sway after `slack-desktop` installation, the error went away though, but the Slack app interface does not show up, thus it seems to be a different bug. Note, however, that `slack-desktop-black-theme-night-mood` and `slack-electron` apps from AUR works as expected.

Sorry for the noise!

**Comment by ErikReider** [comment]
Could you provide the exact error output? Does it work with other notification daemons like mako?

**Comment by ErikReider** [comment]
> Anyway, I suggest to enable [GitHub Discussions](https://github.com/features/discussions), as sometimes users have some questions (like this) and other stuff that are not code/bug related.

Done 👍

**Comment by tukusejssirs** [comment]
> Could you provide the exact error output?

I didn’t save it, all I have saved is below (this is an error what I got):

```bash
Failed to call method: org.freedesktop.DBus.StartServiceByName: object_path= /org/freedesktop/DBus: org.freedesktop.DBus.Error.NoReply: Did not receive a reply. Possible causes include: the remote application did not send a reply, the message bus security policy blocked the reply, the reply timeout expired, or the network connection was broken.
```

> Does it work with other notification daemons like mako?

I didn’t check it (since using SwayNC, I haven’t used `mako`, as I find SwayNC much superior :+1:). I have had installed `slack-desktop-black-theme-night-mood`, then I wanted to try the official `slack-desktop` package, but I got that error and no UI (only app indicator). Then re-logged into Sway and the error went away, but still no UI. Then I tried `slack-electron` and it works as expected. No other debugging I’ve done.

> > Anyway, I suggest to enable [GitHub Discussions](https://github.com/features/discussions), as sometimes users have some questions (like this) and other stuff that are not code/bug related.
> 
> Done +1

Consider converting this issue to discussion (from the sidebar) if you think it is not a bug in your app. :wink:

**Comment by ErikReider** [comment]
> I didn’t check it (since using SwayNC, I haven’t used mako, as I find SwayNC much superior +1).

*blushing_emoji*

> I didn’t save it, all I have saved is below (this is an error what I got):

I'm guessing that it's difficult to test sending of notifications in slack where someone else has to do something for you to get a notification?

But if all other programs on your system works, then it should be a slack issue. Specially if other slack packages work. Does the Flatpak work?

> Consider converting this issue to discussion (from the sidebar) if you think it is not a bug in your app. 😉

Will do :)

**Reply by tukusejssirs** [reply]
> But if all other programs on your system works, then it should be a slack issue. Specially if other slack packages work.

Yeah, now I think it is a non-issue of SwayNC, at least not a direct issue.

The only thing that _might_ be an issue is that fact that after installing an app (`slack-desktop`), it could not (for some reason) connect to the notification server (presumably started by SwayNC, as I got notifications of other apps via SwayNC). There _might_ be an issue there, however, I understand that without a reproduction you can’t do anything about it.

**Comment by Jesse205** [comment]


**Reply by StrongArm88** [reply]
What?

**Reply by tukusejssirs** [reply]
Spam. Why can’t report it? Recently, I’ve seen many spam comments on GitHub. 🤔
